doc/source/serve/deployment-graph/chain_nodes_same_class_different_args.md

```diff
@@ -61,12 +42,13 @@ with InputNode() as dag_input:
 print(ray.get(outputs[-1].execute(0)))
 ```
 
-### Outputs
+## Outputs
 
-The graph will add all nodes weights plus the input (which is 0 in this case)
+The graph will add all nodes weights plus the input (which is 0 in this case).
+0(input) + 0(weight) + 1(weight) + 2(weight) = 3
```

Comment:
our `forward()` implementation is `input + self.weight` , isn't this addition rather than multiplication ? More like 

0 + 0 + 1 + 2 ?

Reply:
oh so the (weight) is annotation of the value